client/apps/game/src/dojo/queries.ts

```diff
@@ -399,3 +401,82 @@ export const getMapFromTorii = async <S extends Schema>(
     false,
   );
 };
+
+export const getQuestTilesFromTorii = async <S extends Schema>(
+  client: ToriiClient,
+  components: Component<S, Metadata, undefined>[],
+  questTileIds: ID[],
+) => {
+  return await debouncedGetQuestTilesFromTorii(client, components as any, questTileIds, () => {});
+};
+
+export const getQuestTilesFromToriiQuery = async <S extends Schema>(
+  client: ToriiClient,
+  components: Component<S, Metadata, undefined>[],
+  questTileIds: number[],
+) => {
+  await getEntities(
+    client,
+    {
+      Composite: {
+        operator: "Or",
+        clauses: questTileIds.map((id) => ({
+          Member: {
```

Comment:
i know there's a few places in the repo where we don't do it but for member clauses can you use the member clause builder like this:
```
AndComposeClause([
      MemberClause("s1_eternum-Tile", "col", "Gte", startCol - range),
      MemberClause("s1_eternum-Tile", "col", "Lte", startCol + range),
      MemberClause("s1_eternum-Tile", "row", "Gte", startRow - range),
      MemberClause("s1_eternum-Tile", "row", "Lte", startRow + range),
    ]).build(),
```

Reply:
Have left for now as it does work currently but if we have time will definitely look to change

Reply:
ok np !